### ISSUE TITLE
Could not listen to port 61324

### ISSUE THREAD

**Issue by NicciZar** [chatty/chatty#406]
**Describe the bug**
I cant connect to my account (first time setup) because the program cant listen to port 61324.
I also cant copy the link but that may be because of the port issue,

**To Reproduce**
[Steps to reproduce the behavior:
Connect twitch account

**Expected behavior**
Being able to log into my twitch.tv account

**Debug Log**

[Include a Chatty debug log: https://chatty.github.io/help/help-report_issue.html#log]
[debug_session.txt](https://github.com/chatty/chatty/files/6051500/debug_session.txt)
[debug0.txt](https://github.com/chatty/chatty/files/6051501/debug0.txt)

**Screenshots/Videos**

![Screenshot 2021-02-26 184231](https://user-images.githubusercontent.com/43537611/109336469-b6dd8b80-7863-11eb-9537-c8e515addf5f.png)

**Additional information (please complete the following information):**

- OS: Windows 10 Professional (English language, location Germany)
- Chatty Version: version 0.14-b6
- Java Version (if not using Standalone): I am using the Windows Setup Installer
- When has the issue first occured (Chatty Version or timeframe): on last Release version and latest beta
- How often does the issue occur (everytime, sometimes): when loggin in

(*Tip:* Use `/appinfo` in Chatty to get some of this information.)
Output:
[18:45] Chatty Version 0.14-b6 Hotkey [Memory] max: 74,532 / 395,968 free: 34,698 [Uptime] 5m 43s [System] Java: 1.8.0_201 (Oracle Corporation / D:\Program Files (x86)\Chatty\runtime) OS: Windows 10 (10.0/x86) Locale: en_GB [Connection] Not connected.

**Comment by NicciZar** [comment]
I now tried to connect my account via the 
'/changetoken {token}'
command inside Chatty.

It partially worked.
I have this:
![image](https://user-images.githubusercontent.com/43537611/109353622-eb117600-787c-11eb-838c-52b09784601e.png)

but it seems like I cant get more Access
I also made sure to take the top link from the Chatty Help site [https://chatty.github.io/help/help-guide2.html]
![image](https://user-images.githubusercontent.com/43537611/109353714-0aa89e80-787d-11eb-8140-fc67038fa898.png)

Maybe these Links have to be updated in order for me to get full access to my twitch account.
Or is there another way for me to get access?

I also tried removing / readding the token via 'Remove Token' then '/tokenchange {token}' into the Chatty chat frame.

'Verify Login' showed me that:
![image](https://user-images.githubusercontent.com/43537611/109353996-696e1800-787d-11eb-803b-2acc30efe034.png)

Thank you
